### PR TITLE
Adding reference to future numpy examples

### PR DESCRIPTION
**Pull request by kesterlester**

### CODE REVIEW

**Comment by tpgillam** [comment]
I fear this actually may be somewhat misleading, since under the hood both versions do actually use numpy - the package does have an unavoidable dependency on numpy.

Secondly, the function I am wrapping at the moment is Asymmetric MT2, not symmetric MT2?

**Comment by kesterlester** [comment]
> I fear this actually may be somewhat misleading, since under the hood both versions do actually use numpy - the package does have an unavoidable dependency on numpy.

If you can find a way of expressing my intention better, which was simply to show a user interacting without explicitly using numpy as part of the UI, then that's all I wanted to show. 

I intended that to contrast with a later example, to be created by you, showing a mass mt2 array calculation, or something like that.  E.g. lots of events all compluted for the same value of chiA but 10 values of chiB efficiently

**Comment by tpgillam** [comment]
OK, understood. I'll get back around to this, as I'd also like to transcribe your good explanation of the three different terms.

I don't know if I can commit to your merge request? Or do I need to fork your merge request and make a meta merge request...>?

**Comment by kesterlester** [comment]
Despite the name/title I gave to this merge request, my concern is less about emphasising (or demphasising) where numpy is used, but more on simply:

1. removing the word "Features" as a heading, and replacing it with "Examples"
2. Being happy that there will be more than one example and so each example needs some kind of sub-heading.

So I don't really mind if you reject the request altogether, but I still recommend you change the word "Features" to "Examples of Usage" (or something like that), and then change the sub heading to something innocuous.

Basically we don't have "features" so that word should go.

**Comment by tpgillam** [comment]
Fair enough - I think I've managed to tweak this appropriately in the pull request (maybe by committing to your branch? I'm really not sure how this is working!)

I'll merge this now.
